### PR TITLE
Adding few updates for ROCm

### PR DESCRIPTION
**Pull request by pruthvistony**
- Handling MacOS and Windows case
- Handling LibTorch case

### CODE REVIEW

**Comment by pruthvistony** [comment]
@seemethere @malfet @jeffdaily @sunway513
Please review

File: assets/quick-start-module.js [hunk 1]
```diff
@@ -243,7 +243,13 @@ function commandMessage(key) {
...
 
     "stable,pip,windows,rocm4.0,python":
-      "<b>NOTE:</b> ROCm is not available on Windows<br />",
+      "<b>NOTE:</b> ROCm is not available on Windows",
+
+    "stable,source,macos,rocm4.0,python":
+      "<b>NOTE:</b> ROCm is not available on MacOS",
+
+    "stable,source,windows,rocm4.0,python":
+       "<b>NOTE:</b> ROCm is not available on Windows",
```
Comment (malfet) [inline assets/quick-start-module.js:252]: All build from source requests should just redirect to github.
```suggestion
```

File: assets/quick-start-module.js [hunk 2]
```diff
@@ -363,7 +369,7 @@ function commandMessage(key) {
       "Download here (Pre-cxx11 ABI): <br/><a href='https://download.pytorch.org/libtorch/nightly/cu111/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/nightly/cu111/libtorch-shared-with-deps-latest.zip</a><br/><br> Download here (cxx11 ABI): <br/><a href='https://download.pytorch.org/libtorch/nightly/cu111/libtorch-cxx11-abi-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/nightly/cu111/libtorch-cxx11-abi-shared-with-deps-latest.zip</a>",
 
     "preview,libtorch,linux,rocm4.0,cplusplus":
-      "<b>NOTE:</b> Conda packages are not currently available for ROCm, please use pip instead<br />",
+      "LibTorch binaries are not yet available for ROCm",
```
Comment (malfet) [inline assets/quick-start-module.js:372]:
```suggestion
      "LibTorch binaries are not yet available for ROCm, please build them from source",
```

File: assets/quick-start-module.js [hunk 3]
```diff
@@ -324,7 +330,7 @@ function commandMessage(key) {
       "pip install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># On MacOS, we provide CPU-only packages, CUDA functionality is not provided",
 
     "preview,pip,macos,rocm4.0,python":
-      "pip install --pre torch torchvision torchaudio -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html<br /># ROCm is not available on MacOS",
+      "<b>NOTE:</b> ROCm is not available on MacOS",
```
Comment (malfet) [inline assets/quick-start-module.js:333]: This change is not necessary, as it follows the common pattern for MacOS platform, where all instruction point to CPU only package and in the comment mention that compute platform is not available

File: assets/quick-start-module.js [hunk 4]
```diff
@@ -375,7 +381,7 @@ function commandMessage(key) {
       "MacOS binaries do not support CUDA. Download CPU libtorch here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip'> https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip </a>",
 
     "preview,libtorch,macos,rocm4.0,cplusplus":
-      "ROCm is not available on MacOS. Download CPU libtorch here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip'> https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip </a>",
+      "<b>NOTE:</b> ROCm is not available on MacOS",
```
Comment (malfet) [inline assets/quick-start-module.js:384]: Please don't change this message

File: assets/quick-start-module.js [hunk 5]
```diff
@@ -387,7 +393,13 @@ function commandMessage(key) {
...
 
     "preview,libtorch,windows,rocm4.0,cplusplus":
-      "<b>NOTE:</b> ROCm is not available on Windows<br />",
+      "<b>NOTE:</b> ROCm is not available on Windows",
+
+    "preview,source,macos,rocm4.0,python":
+      "<b>NOTE:</b> ROCm is not available on MacOS",
+
+    "preview,source,windows,rocm4.0,python":
+      "<b>NOTE:</b> ROCm is not available on Windows",
```
Comment (malfet) [inline assets/quick-start-module.js:402]: Please do not add specific overrides for source package

File: assets/quick-start-module.js (hunk 1 above)
Comment (pruthvistony) [reply assets/quick-start-module.js]: Done.

File: assets/quick-start-module.js [hunk 6]
```diff
@@ -255,7 +261,7 @@ function commandMessage(key) {
       "Download here (Pre-cxx11 ABI): <br/><a href='https://download.pytorch.org/libtorch/cu111/libtorch-shared-with-deps-1.8.0%2Bcu111.zip'>https://download.pytorch.org/libtorch/cu111/libtorch-shared-with-deps-1.8.0%2Bcu11.zip</a><br/><br> Download here (cxx11 ABI): <br/><a href='https://download.pytorch.org/libtorch/cu111/libtorch-cxx11-abi-shared-with-deps-1.8.0%2Bcu111.zip'>https://download.pytorch.org/libtorch/cu111/libtorch-cxx11-abi-shared-with-deps-1.8.0%2Bcu111.zip</a>",
 
     "stable,libtorch,linux,rocm4.0,cplusplus":
-      "LibTorch binaries are not available for ROCm, please build it from source",
+      "LibTorch binaries are not yet available for ROCm",
```
[the remark this answers is not in the document]
Comment (pruthvistony) [reply assets/quick-start-module.js]: Done

File: assets/quick-start-module.js (hunk 2 above)
Comment (pruthvistony) [reply assets/quick-start-module.js]: Done

File: assets/quick-start-module.js (hunk 3 above)
Comment (pruthvistony) [reply assets/quick-start-module.js]: Done

**Review by malfet — APPROVED** [review]

**Pull request merged**